apps/react/demo/src/ui/Example/Provider/Provider.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { type FC, useState, useMemo, useEffect } from "react";
+import { useFormContext, useWatch } from "react-hook-form";
+import Context from "../Context.js";
+import type { ProviderProps, ProviderValue } from "./types.js";
+
+const Provider: FC<ProviderProps> = ({ items = [], children }) => {
+  // Default to the first item if available
+  const [activeExampleIndex, setActiveExampleIndex] = useState(0);
+
+  const { reset } = useFormContext();
+  const formState = useWatch();
+
+  const activeExample = useMemo(
+    () => items[activeExampleIndex],
+    [activeExampleIndex, items],
+  );
+
+  const css = useMemo(
+    () =>
+      Object.fromEntries(
+        activeExample.controls
+          .filter((control) => !control.disabledOutputFormats?.css)
+          .map((control) => {
+            const { name, transformer } = control;
+            const rawVal = formState[name];
+            const val = transformer ? transformer(rawVal) : rawVal;
+            return [name, val];
```

Comment:
For some reason, this hook is not being run when I return `val` (the transformed val) instead of `rawVal`. Needs fix

Reply:
Fixed, I was handling form state resets between examples incorrectly